app/controllers/projects_controller.rb

```diff
@@ -28,13 +28,9 @@ def index
 
   def pay
     @project = Project.find(params[:project_id])
-    if @project.developers.any? {|dev| dev.status == 'unavailable'}
-      flash[:error] = "Some of your developers are unavailable"
-    else
-      @project.update_attribute(:status, 1)
-      @project.developers.each do |dev|
-        dev.update_attribute(:status,'unavailable')
-      end
+    @project.update_attribute(:status, 1)
+    @project.developers.each do |dev|
+      dev.update_attribute(:status,'unavailable')
```

Comment:
Space missing after comma.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.